README.md

```diff
@@ -123,49 +148,29 @@ using System.Collections.Generic;
 
 public class MyCoolBehaviorScript : MonoBehaviour {
     void Start () {
-        Branch.initSession(CallbackWithParams);
```

Comment:
Hey @Acimaz - It's totally fine to use it! I just didn't want to break the builds of partners who have integrated before it existed.